23-Kinetic Loader/style.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #2c3e50;
+  display: flex;
+  margin: 0;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.kinetic {
+  position: relative;
+  height: 80px;
+  width: 80px;
+}
+
+.kinetic::after,
+.kinetic::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 0;
+  height: 0;
+  border: 50px solid transparent;
+  border-bottom-color: #fff;
+  animation: rotateA 2s linear infinite 0.5s;
+}
+
+.kinetic::before {
```

Comment:
cool, nice usage of psudo elements.
this is actually a good topic that I think you should dive in deeper a little bit more:
here is a video: https://www.youtube.com/watch?v=OtBpgtqrjyo 
make sure you are real comfortable with this topic